fpx-workers/src/lib.rs

```diff
@@ -33,14 +41,49 @@ async fn fetch(
 ) -> Result<axum::http::Response<axum::body::Body>> {
     console_error_panic_hook::set_once();
 
+    let env = Arc::new(env);
+    let state = WorkerApiState { env: env.clone() };
+
+    let events = DurableObjectsEvents::new(env.clone());
+    let boxed_events = Arc::new(events);
+
     let d1_database = env.d1("DB").expect("unable to create a database");
 
     let store = D1Store::new(d1_database);
     let boxed_store = Arc::new(store);
-    let events = ServerEvents::new();
 
-    let service = service::Service::new(boxed_store.clone(), events.clone());
-    let mut router = api::create_api(events, service, boxed_store);
+    let service = service::Service::new(boxed_store.clone(), boxed_events.clone());
+    let api_router = api::create_api(boxed_events, service, boxed_store);
+
+    let mut router: axum::Router = axum::Router::new()
+        .route("/api/ws", get(ws_connect))
+        .route("/api/ws/broadcast", post(ws_broadcast))
```

Comment:
This endpoint would be exposed to the public, but it should only be defined on the durable object worker, right?

Reply:
Oops, this was an old testing endpoint! Removed it :)